crates/hotshot/examples/combined/multi-validator.rs

```diff
@@ -58,9 +63,12 @@ async fn main() {
                 NodeImpl,
                 ThisRun,
             >(ValidatorArgs {
-                url,
+                url: args.url,
                 port: args.port,
                 public_ip: args.public_ip,
+                network_config_file: args
+                    .network_config_file
+                    .map(|s| format!("{}-{}", s, node_index)),
             })
```

Comment:
Nit: we could add a `from_multi_args` constructor to `ValidatorArgs` to simplify the code here.

Reply:
I like this idea, will do

Reply:
Done !